blog/forge-v6-release.md

```diff
@@ -0,0 +1,189 @@
+
+## Installing / upgrading to Forge v6
+
+:::info
+
+This section assumes you have a pre-existing project. If you are looking to get started with a new Forge project, head on to the [Getting Started] section in the docs. 
+
+:::
+
+Forge will try to migrate from Electron Forge v5 automatically as much as possible using the same process as importing an existing Electron app to Forge. Some of it may need to be migrated manually. Importing your settings can be done using the [CLI API] node module, which at a high level is a wrapper for the Forge [Core API] that is accessible from the terminal.
+
+<Tabs>
+  <TabItem value="Yarn" label="Yarn" default>
+
+```
+   cd my-app
+   yarn add --dev @electron-forge/cli
+   yarn electron-forge import
+```
+
+  </TabItem>
+  <TabItem value="NPM" label="NPM">
+
+```
+   cd my-app
+   npm install --save-dev @electron-forge/cli
+   npm exec --package=@electron-forge/cli -c "electron-forge import"
+```
+
+  </TabItem>
+</Tabs>
+
+Manual migration details can be found in the Forge [import documentation]. Further help can be offered in our [Discord](https://discord.gg/f4cH9BzaDw).
+
+## Why switch to Forge from an existing pipeline?
+
+If your app already uses an existing build tooling solution that provides packaging and publishing capabilities, the benefits associated with adopting Electron Forge can still outweigh the initial switching cost.
+
+We believe there are two main advantages to using Forge:
+
+1. *Forge receives new features for application building as soon as they are supported in Electron*. This means that alongside the implicit features and plugins that Forge provides, users won't need to wire in new tooling support themselves, or wait for that support to be eventually implemented by other packages before upgrading. This reduction in the burden of maintenance for Forge applications has already been observed when Electron released `@electron/universal` [support](https://github.com/electron/universal) as well as [ASAR Integrity checking](https://www.electronjs.org/docs/latest/tutorial/asar-integrity), which were features that Forge supported out of the box.
+
+1. *Forge's multi-package architecture makes it easy to understand and extend.* Forge was built with the intention of supporting custom plugins, makers and publishers. Since Forge is made up of many smaller packages with clear responsibilities, it is easier to follow code flow. In addition, Forge's extensible API design means that you can write your own additional build logic separate from the provided configuration options for advanced use cases. For more details on extensibility, see the [Extending Electron Forge] section of the docs.
```

Comment:
```suggestion
2. *Forge's multi-package architecture makes it easy to understand and extend.* Forge was built with the intention of supporting custom plugins, makers and publishers. Since Forge is made up of many smaller packages with clear responsibilities, it is easier to follow code flow. In addition, Forge's extensible API design means that you can write your own additional build logic separate from the provided configuration options for advanced use cases. For more details on extensibility, see the [Extending Electron Forge] section of the docs.
```

Reply:
See https://github.com/electron/electronjs.org-new/pull/278#discussion_r1009869856 😄